L1Trigger/L1THGCal/python/hgcalLayersCalibrationCoefficients_cfi.py

```diff
@@ -110,56 +110,14 @@
                                0.328053
                                )
 
-TrgLayer_dEdX_weights = cms.vdouble(0.0,   # there is no layer zero
-                           8.603+8.0675, # Mev
-                           0., 
-                           8.0675*2, 
-                           0., 
-                           8.0675*2, 
-                           0., 
-                           8.0675*2., 
-                           0., 
-                           8.0675+8.9515, 
-                           0., 
-                           10.135*2, 
-                           0., 
-                           10.135*2., 
-                           0., 
-                           10.135*2., 
-                           0., 
-                           10.135*2., 
-                           0., 
-                           10.135+11.682, 
-                           0., 
-                           13.654*2., 
-                           0., 
-                           13.654*2., 
-                           0., 
-                           13.654*2., 
-                           0., 
-                           13.654+38.2005, 
-                           0., 
-                           55.0265, 
-                           49.871, 
-                           49.871, 
-                           49.871, 
-                           49.871, 
-                           49.871, 
-                           49.871, 
-                           49.871, 
-                           49.871, 
-                           49.871, 
-                           49.871, 
-                           62.005, 
-                           83.1675, 
-                           92.196, 
-                           92.196, 
-                           92.196, 
-                           92.196, 
-                           92.196, 
-                           92.196, 
-                           92.196, 
-                           92.196, 
-                           92.196, 
-                           92.196, 
-                           46.098)
+from RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi import dEdX
+
+TrgLayer_dEdX_weights = cms.vdouble()
+for lid, lw in enumerate(dEdX.weights):
+    if lid > 29:
+        TrgLayer_dEdX_weights.append(lw)
+    else:
+        if (lid % 2) == 1:
+            TrgLayer_dEdX_weights.append(lw+dEdX.weights[lid-1])
```

Comment:
why do only even layers get weights? (just my curiosity)

Reply:
Only half of the EE layers are used in the L1 trigger (for cost and power reasons)